arp-speaker/Dockerfile

```diff
@@ -0,0 +1,9 @@
+FROM alpine:latest
```

Comment:
Check out the contents of the `dockerfiles/` directory, it has some templates for dev+prod containers for the metallb binaries. At some point you might want to try using those templates, which might work as well?